internal/dumper/dumper.go

```diff
@@ -300,7 +306,7 @@ func (d *Dumper) dumpTable(conn *Connection, database string, table string) erro
 
 		if (chunkbytes / 1024 / 1024) >= d.cfg.ChunksizeInMB {
 			query := strings.Join(inserts, ";\n") + ";\n"
-			file := fmt.Sprintf("%s/%s.%s.%05d.sql", d.cfg.Outdir, database, table, fileNo)
+			file := fmt.Sprintf("%s/%s.%s.%05d.sql", d.cfg.Outdir, databaseHandle, table, fileNo)
```

Comment:
@mscoutermarsh FYI this is new in the last commit. Seems like it should be harmless at worst, but easy to revert if you foresee issues.